tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -42,26 +43,288 @@ limitations under the License.
 #include "tensorflow/core/util/mkl_util.h"
 
 #ifndef INTEL_MKL_ML
-
 #include "mkldnn.hpp"
 
 using mkldnn::prop_kind;
 using mkldnn::stream;
-
-using mkldnn::convolution_direct;
 using mkldnn::convolution_forward;
+using mkldnn::convolution_direct;
+
 #else
 #include "mkl_dnn.h"
 #include "mkl_dnn_types.h"
 #endif
 
 namespace tensorflow {
 
+#ifndef INTEL_MKL_ML
+
+template <typename T>
+class Conv2DFwd: public DnnOp {
+ public:
+  Conv2DFwd(memory::dims src_dims,
```

Comment:
Could you run clang-format on this file?

Reply:
Done. Also run cpplint utility